src/main/java/duke/Duke.java

```diff
@@ -45,7 +45,7 @@ private static void respondToInput() {
         while (!userInput.equals("bye")) {
             try {
                 if (userInput.equals("list")) {
-                    displayList();
+                    displayList(tasks);
                 } else if (checkIsInputEquals(userInput, "done ")) {
                     Task task = tasks
                             .get(Integer.parseInt(userInput.substring("done ".length())) - 1);
```

Comment:
Perhaps you can consider creating a variable to explain what Integer.parseInt does in this case, so that it is clearer for the users.